src/v/security/audit/schemas/schemas.h

```diff
@@ -176,5 +182,49 @@ class ocsf_base_event : public ocsf_base_impl {
 
         return h;
     }
+
+    /// Method to estimate the size of an ocsf message
```

Comment:
this is estimating the size of the serialized message, correct?  Does this take into account the size of the name of the object or just the value of the object

Reply:
No this is estimating the size of the object in memory, since the message is not yet serialized for the time it exists within the queue.

Reply:
Thanks for the clarification!